src/v2/Components/Footer/FooterDownloadAppBanner.tsx

```diff
@@ -13,12 +13,12 @@ import { Media } from "v2/Utils/Responsive"
 import { DownloadAppBadges } from "../DownloadAppBadges/DownloadAppBadges"
 
 const DESKTOP_COVER_IMAGE = resized(
-  "https://files.artsy.net/images/artsy_app-download-footer_2x_max.jpg",
+  "https://files.artsy.net/images/App Download Banner_1200x2440_2x-1656078840527.png",
```

Comment:
The quality on this bg image isn't really acceptable. Can we locate the original artwork and remake it?

Reply:
![](https://static.damonzucconi.com/_capture/CbrwEcdX.png)

Reply:
Works for me. If you know which artwork it is I can also just recreate it too since they will take forever.

Reply:
I don't have any idea. I'll ask Lois rn.